### ISSUE TITLE
Feature requirment: Support Mosh in WindTerm

### ISSUE THREAD

**Issue by jamesvren** [kingToolbox/WindTerm#1798]
It would be good to see that we can add Mosh support in future release. It can keep connection in some scenario that network are unstable.
Thanks in advance.

**Comment by yankedi** [comment]
Yes, I need mosh,my server latency is around 300ms, which is very frustrating with SSH.
